helm/chart-operator/templates/deployment.yaml

```diff
@@ -43,7 +43,7 @@ spec:
       - key: node-role.kubernetes.io/master
         effect: NoSchedule
       {{ if (.Values.Installation) }}
-      dnsPolicy: Default
+      dnsPolicy: ClusterFirst
```

Comment:
`Default` can't find out the in-cluster service name. So fixing it to `ClusterFirst`
 
```
D 10/29 12:18:48 /apis/application.giantswarm.io/v1alpha1/namespaces/giantswarm/charts/kubernetesd-unique status getting status for release `kubernetesd-unique` | chart-operator/v2/service/controller/chart/resource/status/create.go:31 | controller=chart-operator-chart | event=update | loop=25 | version=207883903
{"caller":"github.com/giantswarm/chart-operator/v2/service/controller/chart/resource/status/create.go:106","controller":"chart-operator-chart","event":"update","level":"error","loop":"25","message":"sending webhook to `http://app-operator-unique.giantswarm.svc.cluster.local:8000/status/giantswarm/kubernetesd-unique` failed","object":"/apis/application.giantswarm.io/v1alpha1/namespaces/giantswarm/charts/kubernetesd-unique","resource":"status","stack":{"annotation":"Patch \"http://app-operator-unique.giantswarm.svc.cluster.local:8000/status/giantswarm/kubernetesd-unique\": dial tcp: lookup app-operator-unique.giantswarm.svc.cluster.local on 168.63.129.16:53: no such host","kind":"unknown","stack":[{"file":"/root/project/service/controller/chart/resource/status/create.go","line":154}]},"time":"2020-10-29T12:18:48.453326+00:00","version":"207883903"}
```